### PR TITLE
RTEMIS-78: Formatted the warning message and fixed an warning.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-78

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Formatted the warning message.

---
### Extras
warning format-2

### CODE REVIEW

**Review by jaswinsingh — COMMENTED** [review]
@kunal-sin Currently we are showing failed UDISE code with individual warning. Can we group udise code which are failing for same reason, this will prevent long warning message if multiple udise code fails.

File: modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php [hunk 1]
```diff
@@ -254,31 +257,41 @@ public static function finishedCallback(bool $success, array $results, array $op
...
+          $errorMessages[] = "$udiseCode: " . implode(', ', $udiseErrors);
+
+          $val .= '<li>' . htmlspecialchars($udiseCode) . '</li>';
+          if (!empty($udiseErrors)) {
+            $val .= '<ul>';
+            foreach ($udiseErrors as $udiseError) {
+              $val .= '<li>' . $udiseError . '</li>';
+            }
+            $val .= '</ul>';
           }
```
Comment (kunal-sin) [inline modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php:276]: Can we use ` '#theme' => 'item_list'`

**Comment by kunal-sin** [comment]
> @kunal-sin Currently we are showing failed UDISE code with individual warning. Can we group udise code which are failing for same reason, this will prevent long warning message if multiple udise code fails.

Make sense
@RohanSingh-inno

File: modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php (hunk 1 above)
Comment (RohanSingh-inno) [reply modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php:276]: yes

**Comment by RohanSingh-inno** [comment]
> > @kunal-sin Currently we are showing failed UDISE code with individual warning. Can we group udise code which are failing for same reason, this will prevent long warning message if multiple udise code fails.
> 
> Make sense @RohanSingh-inno

Yes we can do this. But, managing errors individually might become cumbersome for admins, especially when multiple errors are associated with the same UDISE code. In such cases, UDISE codes with multiple errors may appear under different error headers, leading to repetitive handling tasks. For instance, grouping errors by UDISE code can help here.

**Comment by RohanSingh-inno** [comment]
@jaswinsingh And @kunal-sin We can add a scroll bar from the front end If we move forward with this approach.

**Comment by kunal-sin** [comment]
> > > @kunal-sin Currently we are showing failed UDISE code with individual warning. Can we group udise code which are failing for same reason, this will prevent long warning message if multiple udise code fails.
> > 
> > 
> > Make sense @RohanSingh-inno
> 
> Yes we can do this. But, managing errors individually might become cumbersome for admins, especially when multiple errors are associated with the same UDISE code. In such cases, UDISE codes with multiple errors may appear under different error headers, leading to repetitive handling tasks. For instance, grouping errors by UDISE code can help here.

I think we need a discussion on this. Let's keep it as it is for now. @RohanSingh-inno Add a @todo to update the logic.

**Pull request merged**